scripts/crossVault.ts

```diff
@@ -0,0 +1,67 @@
+import hre from "hardhat";
+import { ethers } from "hardhat";
+import { Contract } from "ethers";
+import { forkNetwork, setupAccount, grantToken, parseToken } from "../utils";
+import fs from 'fs';
+
+async function deployVault(
```

Comment:
we should use hardhat deploy for deployments - it automates a lot of the deployment & tracking tasks

Reply:
i can help migrate this to a deploy script